index.html

```diff
@@ -9,8 +9,32 @@
 		<link rel="stylesheet" href="styles.css">
 	</head>
 
-	<body>
+	<body class="body">
+		<header>
+			<div class="header">
+					<img src="image_geometry_1.svg" alt="geometric image" class="image_header">
+					<img src="ic_menu.svg" alt="menu" class="ic_image">
```

Comment:
Hi @Bavon101 ✋ The hamburger button should have a `nav` tag. Please add a `nav` tag on the hamburger button.  Also, the hamburger is a button or icon, not an image. You can check [this link](https://fontawesome.com/v5.15/icons/bars?style=solid) for a referral to a hamburger icon.